typings/index.d.ts

```diff
@@ -0,0 +1,22 @@
+import * as express from "express"
```

Comment:
So this module works just fine without Express. For example, using the Node.js HTTP server, connect, and more (see the examples in the README). Can the typings not be specific to Express.js, please?

Reply:
Made the more generic now